src/main/kotlin/model/AbstractPlayer.kt

```diff
@@ -0,0 +1,49 @@
+package model
+
+abstract class AbstractPlayer(open val name: PlayerName, val playerType: PlayerType) {
+    private val cardMutableList: MutableList<Card> = mutableListOf()
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-blackjack/pull/43#discussion_r475062468

```suggestion
    private val _cards: MutableList<Card> = mutableListOf()
```

Reply:
수정하였습니다.